instrumentation/opentelemetry-instrumentation-grpc/setup.cfg

```diff
@@ -42,7 +42,7 @@ packages=find_namespace:
 install_requires =
     opentelemetry-api == 0.13dev0
     opentelemetry-sdk == 0.13dev0
-    grpcio == 1.30
+    grpcio ~= 1.27
```

Comment:
Just checking, this isn't required for the test to pass correct?

Reply:
No it is not required. The tests work also without this change.
Since #974 / #975  fixed the tox tests this is just restoring the requirements to the previous state since it now also works with the lower version.

Reply:
Sounds good 👍